apps/yapms/src/lib/components/modals/sharemodal/ShareModal.svelte

```diff
@@ -159,6 +166,19 @@
 				</div>
 			</fieldset>
 
+			<fieldset class="fieldset flex flex-col gap-2">
+				<legend class="fieldset-legend">
```

Comment:
Could you add "gap-1" to the class list here? That class makes the spacing between words follow the normal amount.